engine/table/src/main/java/io/deephaven/engine/table/impl/WhereListener.java

```diff
@@ -39,13 +44,43 @@ class WhereListener extends MergedListener {
     private final WhereFilter[] filters;
     private final ModifiedColumnSet filterColumns;
     private final ListenerRecorder recorder;
-    private final long minimumThreadSize;
     private final boolean permitParallelization;
     private final int segmentCount;
 
     private volatile long initialNotificationStep = NotificationStepReceiver.NULL_NOTIFICATION_STEP;
     private volatile long finalNotificationStep = NotificationStepReceiver.NULL_NOTIFICATION_STEP;
 
+    @ReferentialIntegrity
+    private Runnable delayedErrorReference;
+
+    private static final class DelayedErrorNotifier implements Runnable {
```

Comment:
Same comment on this `DelayedErrorNotifier`; rebase + use the version in `io.deephaven.engine.impl.util`.